### PR TITLE
refactor: request queue endpoints

### LINKED ISSUE #13: Migrate request queues
- [x] file structure -> based on path name, e.g /openapi/paths/actor-builds
- [x] Fix operationId - make it camelCased - e.g GetListOfKeyValueStores
- [x] File name of endpoints -> remove v2@ e.g. actor-tasks@{actorTaskId}@input.yaml

### PR DESCRIPTION
**Pull request by janbuchar**
- **Move request queue path files to a separate directory**
- **Fix operationIds**
- **Move schemas to subfolder**
- **Adjust batch request queue operation responses to reflect reality**
- closes #13

### CODE REVIEW

File: openapi/components/schemas/request-queues/Listrequestsresponse.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+title: Listrequestsresponse
```
Comment (HonzaTuron) [inline openapi/components/schemas/request-queues/Listrequestsresponse.yaml:1]: Would you use PascalCase `ListRequestsResponse` for title and file name instead?

File: openapi/components/schemas/request-queues/Getrequestresponse.yaml
Comment (HonzaTuron) [inline openapi/components/schemas/request-queues/Getrequestresponse.yaml:1]: Missing PascalCase in name

File: openapi/components/schemas/request-queues/Getrequestqueueresponse.yaml
Comment (HonzaTuron) [inline openapi/components/schemas/request-queues/Getrequestqueueresponse.yaml:1]: Missing PascalCase in name

File: openapi/components/schemas/request-queues/Getlistofrequestqueuesresponse.yaml
Comment (HonzaTuron) [inline openapi/components/schemas/request-queues/Getlistofrequestqueuesresponse.yaml:1]: Missing PascalCase in name

File: openapi/components/schemas/request-queues/Getheadresponse.yaml
Comment (HonzaTuron) [inline openapi/components/schemas/request-queues/Getheadresponse.yaml:1]: Missing PascalCase in name

File: openapi/components/schemas/request-queues/Getheadandlockresponse.yaml
Comment (HonzaTuron) [inline openapi/components/schemas/request-queues/Getheadandlockresponse.yaml:1]: Missing PascalCase in name

File: openapi/components/schemas/request-queues/Createrequestqueueresponse.yaml
Comment (HonzaTuron) [inline openapi/components/schemas/request-queues/Createrequestqueueresponse.yaml:1]: Missing PascalCase in name

File: openapi/components/schemas/request-queues/Addrequestresponse.yaml
Comment (HonzaTuron) [inline openapi/components/schemas/request-queues/Addrequestresponse.yaml:1]: Missing PascalCase in name

File: openapi/components/schemas/request-queues/Prolongrequestlockresponse.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+title: Prolongrequestlockresponse
```
Comment (HonzaTuron) [inline openapi/components/schemas/request-queues/Prolongrequestlockresponse.yaml:1]: Missing PascalCase in name and title

File: openapi/components/schemas/request-queues/Updaterequestqueuerequest.yaml
Comment (HonzaTuron) [inline openapi/components/schemas/request-queues/Updaterequestqueuerequest.yaml:1]: Missing PascalCase in name

File: openapi/components/schemas/request-queues/Updaterequestqueueresponse.yaml
Comment (HonzaTuron) [inline openapi/components/schemas/request-queues/Updaterequestqueueresponse.yaml:1]: Missing PascalCase in name

File: openapi/components/schemas/request-queues/Updaterequestresponse.yaml
Comment (HonzaTuron) [inline openapi/components/schemas/request-queues/Updaterequestresponse.yaml:1]: Missing PascalCase in name

File: openapi/components/schemas/request-queues/userData.yaml
Comment (HonzaTuron) [inline openapi/components/schemas/request-queues/userData.yaml:1]: Might be better to start with capital letter in filename?

**Review by HonzaTuron — CHANGES_REQUESTED** [review]

**Comment by janbuchar** [comment]
@HonzaTuron thanks for the detailed review, everything should be resolved now

**Review by HonzaTuron — APPROVED** [review]
Cool, that was fast :)

**Pull request merged**